addons/base_address_extended/tests/test_street_fields.py

```diff
@@ -19,7 +19,9 @@ def test_partner_create(self):
             ['Chaussee de Namur 1', 'Chaussee de Namur', '1', ''],
             ['40 Chaussee de Namur', '40 Chaussee de Namur', '', ''],
             ['Chaussee de Namur, 40 - Apt 2b', 'Chaussee de Namur,', '40', 'Apt 2b'],
-            ['header Chaussee de Namur, 40 trailer', 'header Chaussee de Namur, 40 trailer', '', ''],
+            ['header Chaussee de Namur, 40 trailer ', 'header Chaussee de Namur, 40 trailer', '', ''],
+            ['\nCl 53\n # 43 - 81', 'Cl 53\n #', '43', '81'],
+            ['Street Line 1\nNumber Line 2 44 76', 'Street Line 1\nNumber Line 2 44', '76', '']
```

Comment:
Do add a trailing comma here, that way if we need to add new cases the change won't need to update this line as you had to do here.